index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Montserrat Cruz | Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="header-nav">
+    <a href="#" id="mylogo">My Logo</a>
+    <nav class="navMenu">
+  
+        
+        <div id="menu">
+            <i class="bar"></i>
+            <i class="bar"></i>
+            <i class="bar"></i>
+        </div>
+         
+        
+      </nav>
+
+  </header>
+
+  <main class="flex-container wrapper">
+    <section id="head-container">
+      <h1 class="headers">
+        I'm Montserrat, <br />
+        Glad to see you!
+      </h1>
+      <p class="description">
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience. If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me!
+      </p>
+      <h2 class="connect">LET’S CONNECT</h2>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="https://twitter.com/montsecruzg9" rel="noopener noreferrer" target="_blank">
+            <img src="./assets/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/montserrat-cruz-695594120/" rel="noopener noreferrer" target="_blank">
+            <img src="assets/Linkedin.png" alt="linkedIn" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="assets/medium.png" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="https://github.com/montsecruz9" rel="noopener noreferrer" target="_blank">
+            <img src="assets/github.png" alt="git" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="assets/angel.png" alt="angel" />
+          </a>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] Excellent work 👏. Your HTML code appears to be semantically correct. However, I would recommend a few changes to make your UI design look like the Figma template.

- [ ] First and foremost, there is a left and right margin applied to the **fex-container** and **header-nav** that adds an unwanted gap to the page; I would recommend you fix this to ensure that your design looks as close to the template as possible.

- [ ] Second, the background image is not properly positioned compared to the template there is some part that we can't see. Please ensure to address that too.

### Your design :point_down: 
![Screenshot from 2023-06-02 15-11-28](https://github.com/montsecruz9/personal-portfolio/assets/89228743/3cfb2abf-121d-4635-867f-3139f06ba3a7)

### The template :point_down: 
![Screenshot from 2023-06-02 16-06-14](https://github.com/montsecruz9/personal-portfolio/assets/89228743/8a7b7f71-3f64-45c7-b145-b91c03874669)

### To assist you I am going to provide some suggestions in the CSS file. Please, feel free to apply them. To get the below result with is more close to the template. 😊

### Result :point_down: 
![Screenshot from 2023-06-02 15-10-57](https://github.com/montsecruz9/personal-portfolio/assets/89228743/17b9bb02-701a-4eef-8487-8e1d5558a0bf)

Reply:
You did great work on implementing the changes required by the previous reviewer, but there are some changes that still need to fix to make your project complete.

Kindly I suggest you stick to the Figma design as much as possible to make your project look more professional.

> ## The Figma Design
>
><img width="387" alt="Screenshot 2023-06-03 at 7 16 15 PM" src="https://github.com/montsecruz9/personal-portfolio/assets/91237525/ed78a671-ea3c-41d4-9a8e-429960487d7d">
>
> ## Your Design
>
><img width="373" alt="Screenshot 2023-06-03 at 7 12 19 PM" src="https://github.com/montsecruz9/personal-portfolio/assets/91237525/8460fccc-54d0-46cd-b908-8be4fa42eaba">